src/service/resources-impl.js

```diff
@@ -1272,6 +1313,23 @@ export class Resources {
       }
     }
   }
+
+  /**
+   * Removes the task resource from the discovered first viewport
+   * resources array.
+   * @param {!TaskDef} task
+   */
+  checkForFirstViewportElementsWhenTaskDone_(task) {
```

Comment:
I agree. I think we need to be very conscious of internal variables in this class.